lightning-background-processor/src/lib.rs

```diff
@@ -360,12 +376,24 @@ macro_rules! define_run_body {
 				break;
 			}
 
+			if $timer_elapsed(&mut last_forwards_processing_call, cur_batch_delay) {
+				$channel_manager.get_cm().process_pending_htlc_forwards();
```

Comment:
I was also wondering whether this function needs to be called twice per htlc (add and fail/settle), and incurs hits the delay twice? And whether it is also required to be called at the sender and at the receiver node? (I know in lnd that the abstraction was so that for example on the final hop, it would 'forward' to the invoice)